modules/mpv/meta.nix

```
@@ -1,4 +1,7 @@
 { lib, ... }:
 {
-  maintainers = [ lib.maintainers.naho ];
+  maintainers = with lib.maintainers; [
+    naho
+    awwpotato
```

We might want to keep this sorted:

```suggestion
    awwpotato
    naho
```

Not sure what the Nixpkgs convention on this is, if it has any.